lib/modules/versioning/node/readme.md

```diff
@@ -1,3 +1,5 @@
-Renovate's Node.js versioning is a wrapper around npm's versioning, except that it makes sure to strip "v" prefixes from exact versions when replacing.
+Renovate's Node.js versioning is a wrapper around npm's versioning.
+But Renovate removes any `v` prefixes from exact versions when replacing.
 
-It is planned to extend it one day to support "stability" awareness, because Node.js's version stability does not follow the SemVer approach.
+We plan to extend the Node.js versioning to support "stability" awareness.
+This is nice to have because Node.js's version stability does not match the SemVer rules.
```

Comment:
Is there any issue we can link to here for the plan to extend Node.js versioning to support "stability" awareness? I searched a bit, but could not find a 1 to 1 match. 😄

Reply:
I couldn't find an issue for Node.js stability specifically. The closest I think matches is https://github.com/renovatebot/renovate/issues/5990

Reply:
I think I found the source commit for the `node/readme.md` file. Here's a quote from that commit: [^1]

> ## Implementation plan/status
> 
> - [x] Strip v prefix
> - [ ] Support Node.js-specific "stable" awareness

Multiple files in that commit have a section like this. So I think the whole "We plan to extend ..." stuff was a developer todo-list, instead of something that should go in the end-user documentation.

> I couldn't find an issue for Node.js stability specifically. The closest I think matches is #5990

I don't know if we need a new issue just for "Node.js stability awareness" or if we should link to #5990 in the docs. I'll let @rarkins decide what to do here. 😉 

[^1]: https://github.com/renovatebot/renovate/commit/e5cd1cd4b0beace2d1f88280ca2bc66d9ce2a752#diff-ecd41b3a4fd2acb98436c3d49e330ec141e1b18875c5b8bd618d3428546c9df8

Reply:
```suggestion
```